addons/mail/static/src/components/discuss/tests/discuss_moderation_tests.js

```diff
@@ -1227,6 +1227,130 @@ QUnit.test('new pending moderation message posted by someone else when moderator
     );
 });
 
+QUnit.test('accept multiple moderation messages', async function (assert) {
```

Comment:
I would split test in 2, because it's a bit too long IMO. One for partial accept (2 out of 4 messages), and one for being able to check + accept message again after accepting some messages before (which I assume is the behaviour that this PR should fix?)

Reply:
Idea is 2nd test require much less assertions, because these are covered by other test. And also the name of test means precisely what is being tested.

Reply:
not sure to understand the benefit and not sure to understand what you expected but I made two tests, hopefully it's what you meant

Reply:
Basically purpose is to have much precise intentions of tests, so the name is almost enough to get what it does. The more generic the name of test, the longer/complex/weird its code usually becomes.

Reply:
Was it pushed? I don't see 2 tests